src/app/src/main/java/com/uphold/uphold_android_sdk/model/Transaction.java

```diff
@@ -207,7 +207,7 @@ private String getCardIdPath() {
         }
 
         if (transactionCommitRequest == null) {
-            transactionCommitRequest = new TransactionCommitRequest(null);
+            transactionCommitRequest = new TransactionCommitRequest(null, null);
```

Comment:
Do we need to change this?

Reply:
Yeah, overload constructor: `TransactionCommitRequest(String)` and `TransactionCommitRequest(Beneficiary)`.